### ISSUE TITLE
Something looks wrong with 'dot'

### ISSUE THREAD

**Issue by FredM67** [DenverCoder1/doxygen-github-pages-action#16]
Even graphviz is installed, the action is complaining about 'dot'.
```Generating caller graph for function pairForceLoad::getDuration
sh: 1: dot: not found
error: Problems running dot: exit code=127, command='dot', arguments='"/home/runner/work/PVRouter-3-phase/PVRouter-3-phase/docs/html/class__rg__OffsetForce_a2d0fe1a08e4856f38a91f26b46066733_cgraph.dot" -Tpng -o "/home/runner/work/PVRouter-3-phase/PVRouter-3-phase/docs/html/class__rg__OffsetForce_a2d0fe1a08e4856f38a91f26b46066733_cgraph.png" -Tcmapx -o "/home/runner/work/PVRouter-3-phase/PVRouter-3-phase/docs/html/class__rg__OffsetForce_a2d0fe1a08e4856f38a91f26b46066733_cgraph.map"'```

**Comment by DenverCoder1** [comment]
Try with version 1.2.0.

You may have been using 1.1.0 which did not install graphviz by default.

**Comment by FredM67** [comment]
Fix with version 1.2.0.
That's strange, because in the log of V1.1.0, I did see graphviz installed !!
Never mind... works with 1.2.0.
